worker.js

```diff
@@ -0,0 +1,56 @@
+var Sway = require('sway')
```

Comment:
might not have time for karma (tonight)... but we'll get there :stuck_out_tongue: